test/cli.test.js

```diff
@@ -4,282 +4,306 @@ process.env.TZ = 'UTC'
 
 const path = require('node:path')
 const { spawn } = require('node:child_process')
-const { test } = require('tap')
+const { test } = require('node:test')
+const { once } = require('./helper')
 
 const bin = require.resolve(path.join(__dirname, '..', 'bin.js'))
 const epoch = 1522431328992
 const logLine = '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo"}\n'
 const env = { TERM: 'dumb', TZ: 'UTC' }
 const formattedEpoch = '17:35:28.992'
 
-test('cli', (t) => {
-  t.test('does basic reformatting', (t) => {
+test('cli', async (t) => {
```

Comment:
Please use describe and remove the await

Reply:
Done